CONTRIBUTING.md

```diff
@@ -4,55 +4,30 @@ Thank you for your interest in contributing to Ice!
 
 ## How to Contribute
 
-Follow these steps to contribute code or documentation to this project:
+To contribute code or documentation to this project, please follow these steps:
 
-1. Download and review carefully our [Contributor Agreement](https://zeroc.com/download/ZeroCContributorAgreement.pdf), and send us a completed and signed copy.
+1. Review the [Contributor Agreement](https://gist.github.com/zcabot/1a4c24dca55adaa83d78cdeabc63226b).
+   You will be prompted by the CLAassistant bot to sign the agreement when you
```

Comment:
I would put a space between CLA and assistant.

Reply:
I'd find something like:
The CLA assistant bot will ask you to sign this agreement when you submit your first pull request.

easier to understand.